src/ui/routes/package/deploy/+page.svelte

```diff
@@ -0,0 +1,70 @@
+<!-- 
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+ -->
+<script lang="ts">
+	import { goto } from '$app/navigation';
+	import { page } from '$app/stores';
+	import { Packages } from '$lib/api';
+	import { pkgStore } from '$lib/store';
+	import { PackageErrNotFound, Spinner } from '$lib/components';
+
+	enum LoadingStatus {
+		Loading,
+		Success,
+		Error
+	}
+	let status: LoadingStatus = LoadingStatus.Loading;
+	let errMessage: string = '';
+	const pkgPath = $page.url.searchParams.get('path');
+	let pkgName: string;
+
+	if (pkgPath === null) {
+		errMessage = 'No package path provided';
+		status = LoadingStatus.Error;
+	} else {
+		pkgName = encodeURIComponent(
+			pkgPath
+				?.split('/')
+				.at(-1)
+				?.replaceAll('zarf-package-', '')
+				.replaceAll('.tar', '')
+				.replaceAll('.zst', '') ?? ''
+		);
+
+		if (pkgPath === 'init') {
+			Packages.findInit()
+				.catch(async (err: Error) => {
```

Comment:
why is the catch callback async? There is nothing async about that and adds a needless wrapper.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch

Also I notice here we're doing `promise.catch.then` and below `promise.then.catch`, recommend we stay consistent (and I usually see `promise.then.catch.finally` as seen here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally.